### ISSUE TITLE
version 8.4.0 -- Error: Cannot find module '@firebase/app-exp'

### ISSUE THREAD

**Issue by lantanabloom** [firebase/firebase-js-sdk#4776]
```
{ Error: Cannot find module '@firebase/app-exp'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (.../node_modules/@firebase/database/dist/index.node.cjs.js:8:14)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3) code: 'MODULE_NOT_FOUND' }

```
Getting this error only in new 8.4.0 version after requiring firebase like so:
```
var firebase = require("firebase/app");
require("firebase/auth");
require("firebase/database");
```

**Comment by dmrwebdev** [comment]
Same issue when using `import 'firebase/database'`, dropping down to v8.3.3 solved the issue immediately.

`./node_modules/@firebase/database/dist/index.esm.js
Module not found: Can't resolve '@firebase/app-exp' in '/home/derek/projects/test2/node_modules/@firebase/database/dist'`

**Comment by Gameplayit17** [comment]
I have the same problem:

```
internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module '@firebase/app-exp'
Require stack:
- /home/node/node_modules/@firebase/database/dist/index.node.cjs.js
- /home/node/node_modules/firebase/dist/index.node.cjs.js
- /home/node/index.js
- /etc/375671974594215936m663393061618843650.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at Module.newFunc (/etc/node_modules/appmetrics/lib/aspect.js:101:26)
    at Module.newFunc [as require] (/etc/node_modules/ibmapm-embed/appmetrics-zipkin/lib/aspect.js:101:26)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.\u003canonymous\u003e (/home/node/node_modules/@firebase/database/dist/index.node.cjs.js:8:14)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/node/node_modules/@firebase/database/dist/index.node.cjs.js',
    '/home/node/node_modules/firebase/dist/index.node.cjs.js',
    '/home/node/index.js',
    '/etc/375671974594215936m663393061618843650.js'
  ]
}
```

**Comment by Feiyang1** [comment]
It is now fixed in 8.4.1.
